cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-lab-overview.adoc

```diff
@@ -0,0 +1,279 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-lab-overview"]
+= Tutorial: Deploying an application
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-intro
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 22-JAN-2024
+
+== Lab overview
+
+=== Lab resources
+
+* source code for the OSToy application: link:https://github.com/openshift-cs/ostoy[]
```

Comment:
s/source/Source code...

I would also put the text into the link boxes so they are linked text.

Reply:
Updated!